examples/reflection/reflection_types.rs

```diff
@@ -105,6 +105,10 @@ fn setup() {
         // This exposes "map" operations on your type, such as getting / inserting by key.
         // Map is automatically implemented for relevant core types like HashMap<K, V>
         ReflectRef::Map(_) => {}
+        // `Set` is a special trait that can be manually implemented (instead of deriving Reflect).
+        // This exposes "set" operations on your type, such as getting / inserting by value.
+        // Set is automatically implemented for relevant core types like HashSet<K>
```

Comment:
Nit: I think std uses `T` instead of `K` in its documentation
```suggestion
        // Set is automatically implemented for relevant core types like HashSet<T>
```